packages/console/src/pages/UserDetails/components/UserConnectors.tsx

```diff
@@ -0,0 +1,124 @@
+import { Languages } from '@logto/phrases';
+import { ConnectorDTO, Identities } from '@logto/schemas';
+import { Optional } from '@silverhand/essentials';
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import useSWR from 'swr';
+
+import Button from '@/components/Button';
+import ImagePlaceholder from '@/components/ImagePlaceholder';
+import UnnamedTrans from '@/components/UnnamedTrans';
+import useApi, { RequestError } from '@/hooks/use-api';
+
+import * as styles from './UserConnectors.module.scss';
+
+type Props = {
+  userId: string;
+  connectors: Identities;
+  onDelete?: (connectorId: string) => void;
+};
+
+type DisplayConnector = {
+  id: string;
+  userId?: string;
+  logo: string;
+  name: Record<Languages, string>;
+};
+
+const UserConnectors = ({ userId, connectors, onDelete }: Props) => {
+  const api = useApi();
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const { data, error } = useSWR<ConnectorDTO[], RequestError>('/api/connectors');
+  const isLoading = !data && !error;
+
+  const handleDelete = async (connectorId: string) => {
```

Comment:
A successful toast is needed?

Reply:
I don't think we need a toast here, because this line will immedietly disappear after deleting. To many toast is "annoying".